Automattic/wp-calypso
# My Home: Add a Quick Start Session card.

This PR adds a Quick Start Sessions card to My Home.

Not Booked | Booked
--- | ---
![image](https://user-images.githubusercontent.com/349751/80265145-65fd7500-864b-11ea-8779-7efcbb0c27e2.png) | ![image](https://user-images.githubusercontent.com/349751/80265182-8b8a7e80-864b-11ea-86f5-fa2327ef6c85.png)

#### Testing instructions

* Go to `/home/:site` for a site with either an eCommerce or Business plan.
* Verify the "Book a session" button goes to `/me/concierge/:site/book`.
* On prod, take a test site with a Business plan and register for a Quick Start session at the above booking URL. (Be sure to comment that we're just testing. 🙃)
* Back on this branch for the Business test site, verify that the card title changes appropriately.
* Verify the "View details" button goes to `/me/concierge/:site/book`.
* Verify the "Reschedule" link goes to `/me/concierge/:site/:sessionId/cancel`.

Fixes #38998
